### PR TITLE
Finish calculator component

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch I did:
A React component for a calculator UI;
Class-based components;
Checked if everything was rendering properly

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          <button className="orange" type="button">
+            =
+          </button>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Calculator;
```
Comment (t-yanick) [inline src/components/Calculator.js:78]: - [ ] Please your app is still rendering the `boiler plate`, created in the last milestone, Please kindly update your `App.js` file and import  your `Calculator component` into it and make sure it is rendering as specified in the design. Checkout the screenshot below for more clarifications. `Remember that App.js component is class-based`

classbased

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @lisandroseia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and under the review._

- [ ] Please kindly `Delete all the boilerplate from CRA (text, images, styles)` as this will make your repository less crowded. Checkout your `public directory` and kindly fix that. 
- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (lisandroseia) [reply src/components/Calculator.js:78]: hi @t-yanick, I can't figure out what this one is about, I exported and imported the class, and everything is a class-based component, I will be very pleased if you could explain me what the problem is :face_with_spiral_eyes:

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @lisandroseia, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**